e2e/app/data-page.ts

```diff
@@ -29,4 +37,40 @@ export default class DataPage extends AuthenticatedPage {
     }
   }
 
+  async selectWorkspaceAction(action: WorkspaceAction) {
+    const ellipsisMenu = new EllipsisMenu(this.page, './/*[@data-test-id="workspace-menu-button"]');
+    switch (action) {
+    case WorkspaceAction.DELETE:
+      await ellipsisMenu.delete();
```

Comment:
Another way to do this would be just to pass the action to ellipsisMenu and  call this.selectAction

Reply:
👍👍